wcfsetup/setup/db/install.sql

```diff
@@ -2216,8 +2216,11 @@ INSERT INTO wcf1_user_group_option_value (groupID, optionID, optionValue) VALUES
 INSERT INTO wcf1_user_group_option_value (groupID, optionID, optionValue) VALUES (4, 3, '1');	-- Administrators
 
 -- default update servers
+-- TODO: change 2019 servers to `online`
```

Comment:
Seriously 6 servers meaning 6 single requests per list-update? That's aweful - otherwise it's not my server.

Reply:
It should be only one update server type using the API version number.

Reply:
The servers for `vortex` and `tornado` will be removed later in the development cycle, they remain for testing purposes to validate that the update server API remains compatible with any change being made.